### ISSUE TITLE
Column type is not properly saved and loaded in GUI data format manager

### ISSUE THREAD

**Issue by pklimai** [mipt-npm/jinr-database-tools#2]
When JSON schema created in data format manager is saved, all "type" values are written as "float" regardless of what was chosen in GUI drop-down.

When program is restarted, the type is instead loaded as "boolean" for all columns.

**Comment by Zelenyy** [comment]
Yes, creation schema with GUI don't finished, because first we should finalize schema format.
